index.js

```diff
@@ -0,0 +1,135 @@
+                  <h3 class="speaker-name">${speakers[0].name}</h3>
+                  <p class="speaker-desc">${speakers[0].description}
+                  </p>
+                  <p>Holds a masters in Divinity from Afric Inland University.</p>
+                </div>
+              </card>
+
+               <card class="speaker first">
+                <img src=${speakers[1].image} alt="Speaker" class="speaker-img">
+                <div class="speaker-info">
+                  <h3 class="speaker-name">${speakers[1].name}</h3>
+                  <p class="speaker-desc">${speakers[1].description}
+                  </p>
+                  <p>Holds a masters in Divinity from Afric Inland University.</p>
+                </div>
+              </card>
+            </div>
+            <div class="speakers">
+               <card class="speaker first">
+                <img src=${speakers[2].image} alt="Speaker" class="speaker-img">
+                <div class="speaker-info">
+                  <h3 class="speaker-name">${speakers[2].name}</h3>
+                  <p class="speaker-desc">${speakers[2].description}
+                  </p>
+                  <p>Holds a masters in Divinity from Afric Inland University.</p>
+                </div>
+              </card>
+
+               <card class="speaker first">
+                <img src=${speakers[3].image} alt="Speaker" class="speaker-img">
+                <div class="speaker-info">
+                  <h3 class="speaker-name">${speakers[3].name}</h3>
+                  <p class="speaker-desc">${speakers[3].description}
+                  </p>
+                  <p>Holds a masters in Divinity from Afric Inland University.</p>
+                </div>
+              </card>
+            </div>
+            <div class="speakers">
+
+
+              <card class="speaker first">
+                <img src=${speakers[4].image} alt="Speaker" class="speaker-img">
+                <div class="speaker-info">
+                  <h3 class="speaker-name">${speakers[4].name}</h3>
+                  <p class="speaker-desc">${speakers[4].description}
+                  </p>
+                  <p>Holds a masters in Divinity from Afric Inland University.</p>
+                </div>
+              </card>
+
+               <card class="speaker first">
+                <img src=${speakers[5].image} alt="Speaker" class="speaker-img">
+                <div class="speaker-info">
+                  <h3 class="speaker-name">${speakers[5].name}</h3>
+                  <p class="speaker-desc">${speakers[5].description}
+                  </p>
+                  <p>Holds a masters in Divinity from Afric Inland University.</p>
+                </div>
+              </card>
+            </div>
+
+            <div class="button-container">
+              <button>More<span class="material-icons material-icons-outlined arrow">
+                  keyboard_arrow_down
+                </span></button>
+
+            </div>
+`;
```

Comment:
- [ ] Good job implementing the way to add the elements on the page, now, let's try to do it through an iteration (a loop, like for, foreach as an example). 🐸